packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -39,6 +39,8 @@ class CupertinoScrollbar extends StatefulWidget {
   const CupertinoScrollbar({
     Key key,
     @required this.child,
+    this.dynamicNavBarPersistentHeight,
```

Comment:
Ya sorry, we shouldn't do this. We shouldn't make any widget's behavior explicitly bound to another or make the nav bar special here. We should use the Sliver system so that non CupertinoScrollbars can easily get notifications for sticky slivers and non CupertinoSliverNavigationBars can report sticky height to scrollbars etc.